src/utils/ColorUtils.js

```diff
@@ -32,13 +32,23 @@ define(function (require, exports, module) {
     "use strict";
     
     /**
+     * Sorted array of all the color names in the CSS Color Module Level 3 (http://www.w3.org/TR/css3-color/)
+     * and "rebeccapurple" from CSS Color Module Level 4
```

Comment:
Minor nit, does not totally need fixing: I'd reword this to 

> Sorted array of all the color names in the CSS Color Module Level 3 and 4 (http://www.w3.org/TR/css3-color/).

I don't think we need to specifically state what color(s) from CSS4 is supported, as ideally we'd support all of them.